tests/test_upload.py

```diff
@@ -90,9 +103,9 @@ def test_get_config_old_format(tmpdir):
         """))
 
     try:
-        upload.upload(dists="foo", repository="pypi", sign=None, identity=None,
-                      username=None, password=None, comment=None,
-                      sign_with=None, config_file=pypirc)
+        kwargs = default_kwargs.copy()
```

Comment:
yes but if we don't extract default_kwargs we will, sooner or later, have to keep changes that test on every argument we add/change